### ISSUE TITLE
Mobile data usage always remains at zero, despite being used. 

### ISSUE THREAD

**Issue by BlueSapphire93** [fakemanoan/LineageOS-Releases#75]
### Build used

lineage-20.0-20240221-UNOFFICIAL-zerofltexx

### Model number

G920F

### Describe the bug

When I switch off my wifi, and browse the web using my mobile data, my data usage still reports that 0 bytes has been used.  

### To Reproduce

1. Disable WIFI
2. Enable mobile data 
3. Open browser and open several different webpages and verify the pages load. 
4. Under Settings -> Network and internet -> Internet -> Mobile carrier name, the data usage remains at zero.
5. 

### Any extra info

Just going off memory, but I though this was working with the release from November. 
First noticed it not working with the release from January.

**Comment by fakemanoan** [comment]
Thanks for the bug report.

This feature never worked. The reason is because our kernel is on 3.10. This means we do not support eBPF, which is exclusively used to measure network traffic in Android 13 onwards. It is a known quirk of being on Android 13 on a legacy kernel. So you can't monitor Wi-Fi traffic, and in this case mobile data traffic too.

The way to fix this is to add eBPF to the 3.10 kernel, which is very hard to do, or update the kernel to 4.4 or newer, which is even harder. 

Basically, it will remain broken for the time being. If you absolutely need this feature, I would suggest using Android 12L, where the feature works fine.

**Comment by BlueSapphire93** [comment]
Thanks for the very quick and clear response.
